packages/documentation-site/patternfly-docs/patternfly-docs.config.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
   hasVersionSwitcher: true,
   hasDesignGuidelines: true,
   hasDarkThemeSwitcher: true,
+  hasRTLSwitcher: true,
```

Comment:
just a reminder to take this out prior to merge

Reply:
Done - thanks!